nuclio/triggers.py

```diff
@@ -124,20 +124,24 @@ def __init__(self, interval="", schedule="", body="", headers=None):
 
 
 class KafkaTrigger(NuclioTrigger):
-    kind = "kafka"
+    kind = "kafka-cluster"
 
-    def __init__(self, url, topic, partitions=None):
+    def __init__(self, brokers, topics, partitions=None, consumer_group="kafka", initial_offset="earliest"):
         super(KafkaTrigger, self).__init__({
             "kind": self.kind,
-            "url": url,
-            "attributes": {"topic": topic},
+            "maxWorkers": 1,
+            "attributes": {"Topics": topics, "Brokers": brokers, "ConsumerGroup": consumer_group,
```

Comment:
as a practice with passing attributes to nuclio, you would want to use `camelCase`

so instead of `ConsumerGroup`, use `consumerGroup`

Reply:
that doesn't work. if it's not upper case it fails to find the attribute

Reply:
alrighty then, i guess because they havent been exposed